client/app/auth/authController.js

```diff
@@ -1,5 +1,16 @@
-angular.module('throwcast.auth', [])
+angular.module('throwcast.auth', ['ngRoute'])
 
+.config(function ($routeProvider) {
```

Comment:
The config really doesn't have anything to do with the auth controller.

I would move the config and the module creation and dependency declaration into its own file (probably `auth.js` in the same folder). Then use `angular.module('throwcast.auth')` to get the created module in this file.